airbyte-webapp/src/config/frequencyConfig.ts

```diff
@@ -1,6 +1,6 @@
-import { ConnectionSchedule } from "core/request/AirbyteClient";
+import { ConnectionScheduleDataBasicSchedule } from "core/request/AirbyteClient";
 
-export const frequencyConfig: Array<ConnectionSchedule | null> = [
+export const frequencyConfig: Array<ConnectionScheduleDataBasicSchedule | null> = [
```

Comment:
```suggestion
export const frequencyConfig: (ConnectionScheduleDataBasicSchedule | null)[] = [
```
To match our style guide :)